docs/content/tutorials/tutorial-add-radius/index.md

```diff
@@ -258,7 +258,7 @@ You will now add Radius to the Guestbook application's Kubernetes deployment man
 Now that Radius has been enabled for your application, run this command again:
 
 ```bash
-rad app connections -a demo
+rad app graph -a demo -g default-demo
```

Comment:
Shd we change rad app connections to rad app graph in all the docs that references connection ? I don't see a CLI reference page for rad app graph in our docs.

Reply:
even though it's a bit beyond the scope of this PR, I agree with your observation and committed a "rebrand" of `rad app connections` to `rad app graph` -- please take a look.

Reply:
The CLI reference pages shouldn't be manually edited. We can't update these until we complete the migration of the CLI code from connections to graph

Reply:
I think its better to update this once rebranding feature is complete ?